apps/nextcloud/templates/services/app/deploy.yml.j2

```diff
@@ -75,4 +79,8 @@ spec:
           defaultMode: 420
           name: nextcloud-app-{{ deployment_stamp }}
       - name: nextcloud-config
-        emptyDir: {}  # volume that lives as long as the pod lives      
+        # volume that lives as long as the pod lives
+        emptyDir: {}
+      securityContext:
+        runAsUser: {{ container_uid }}
```

Comment:
I don't know the reason why `nexcloud_user_id` was used before.
Won't that be a problem to use another UID to run the container @lunika?

Reply:
The `nextcloud` Docker image is built for a specific user id.
I changed the parameter name to match what we now do on other apps and set it to the value that was previously set for `container_uid`

Reply:
Ok. Maybe you can rename the variable  in `apps/nextcloud/vars/all/main.yml` (`nextcloud_user_id` -> `container_uid`) and add a comment to specify that this container is built for a specific user id?

Reply:
<img src="https://user-images.githubusercontent.com/1427165/122026020-256e3700-cdca-11eb-9243-22852f680eea.jpg" width="200"/>

Reply:
@madmatah I didn't rename it but removed it as it is already defined in `groups_vars/all/main.yml` for all applications. Can you check the comment?

Reply:
The default value for `container_uid` defined in `groups_var/all/main.yml` is set to `10000`.
I don't think it will work out of the box, since the nextcloud image is built with UID `1000` according to your comment.

That's why I suggest to override the value for this arnold app.

Reply:
no no my comment should read `10000` as I was talking about the default `container_uid` value. The comment asks to build your image with the uid you want... we have our own image build but this is our problem.

Reply:
Fixed.

Reply:
Oh, ok, It's clear now!
:ship: